app/assets/stylesheets/darkswarm/_shop-product-rows.css.scss

```diff
@@ -9,44 +9,13 @@
         .columns {
           padding-top: 0em;
           padding-bottom: 0em;
-          display: table;
           line-height: 1.1;
-
-          // outline: 1px solid red
-
-          @media all and (max-width: 768px) {
-            font-size: 0.875rem;
-          }
-
-          @media all and (max-width: 640px) {
-            font-size: 0.75rem;
-          }
-        }
-
-        .table-cell {
-          display: table-cell;
-          vertical-align: middle;
-          height: 37px;
         }
       }
 
       // ROW VARIANTS
       .row.variants {
-        margin-left: 0;
-        margin-right: 0;
-        background-color: #ECECEC;
-
-        &:hover, &:focus, &:active {
-          background-color: $clr-brick-light;
-        }
-
-        &:nth-of-type(even) {
-          background-color: #f9f9f9;
-
-          &:hover, &:focus, &:active {
-            background-color: $clr-brick-ultra-light;
```

Comment:
this makes $clr-brick-ultra-light in the branding css file dead code, can be deleted as well.